test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -103,6 +103,7 @@ declare module 'test-case-reporting' {
   }
 
   namespace KarmaReporter {
+    // TODO(rafer45): Add job & build info to the karma report
```

Comment:
Since the job and build are part of Travis, not Karma, they'd be added as a separate interface outside of this Karma namespace.

Reply:
Done.